src/ID3Frames.js

```diff
@@ -545,3 +545,31 @@ module.exports.COMR = {
         return tag
     }
 }
+
+module.exports.GEOB = {
+    create: (data) => {
+        if (!(data instanceof Array)) {
+            data = [data]
+        }
+        const encoding = 3 // UTF-8
+        return Buffer.concat(data.map((geob) => {
+            return new ID3FrameBuilder("GEOB")
+                .appendStaticNumber(encoding)
+                .appendNullTerminatedValue(geob.mimeType ? geob.mimeType : '')
+                .appendNullTerminatedValue(geob.filename ? geob.filename : '', encoding)
+                .appendNullTerminatedValue(geob.description ? geob.description : '', encoding)
+                .appendStaticValue(geob.data)
```

Comment:
To stay as close as possible to the specification, I think we should name this property at least `object` (`encapsulatedObject` is probably an overkill).